wazo_sysconfd/plugins/dhcp_update/http.py

```diff
@@ -1,13 +1,19 @@
-# Copyright 2011-2022 The Wazo Authors  (see the AUTHORS file)
+# Copyright 2022 The Wazo Authors  (see the AUTHORS file)
 # SPDX-License-Identifier: GPL-3.0-or-later
 
+from fastapi import APIRouter
 import subprocess
-from xivo.http_json_server import register, CMD_R, HttpReqError
+
+from wazo_sysconfd.exceptions import HttpReqError
+
+router = APIRouter()
+
 
 DHCPD_UDPATE_COMMAND = ['dhcpd-update', '-dr']
```

Comment:
The `http.py` module should only care about the HTTP-side of the request. The actual work should be done in a `service.py` module that is independent of the HTTP protocol.

Reply:
moved as suggested. thx.